selfdrive/ui/qt/home.cc

```diff
@@ -174,6 +174,7 @@ OffroadHome::OffroadHome(QWidget* parent) : QFrame(parent) {
 }
 
 void OffroadHome::showEvent(QShowEvent *event) {
+  locale_name = QString::fromStdString(Params().get("LanguageSetting")).replace("main_", "");
```

Comment:
Can we put this in the UI state so we just need to read the param once on startup?